src/boi/boi_wifi.cpp

```diff
@@ -293,27 +244,25 @@ void boi_wifi::enter_safe_mode_with_networking(const OptionsStruct *Options){
     delay(100);
 
     WiFi.enableSTA(true);
-    if(!WiFi.mode(WIFI_STA))
-    {
+    if(!WiFi.mode(WIFI_STA)) {
         Serial.println("Failed to set wifi mode");
     }
 
     yield();
     delay(100);
-
     wl_status_t status;
     status = WiFi.begin(Options->SafeModeWifiName,Options->SafeModeWifiPassword);
-    int counter =0;
+    int counter = 0;
 
-    //esp32 appears to have a serious issue about connecting first time around, we force an immediate 2nd cycle hence the begin above
-    //then not incrementing counter until the end
+        //esp32 appears to have a serious issue about connecting first time around, we force an immediate 2nd cycle hence the begin above
+        //then not incrementing counter until the end
     while (status != WL_CONNECTED) {
         delay(100);
         status = WiFi.status();
-        if(status == WL_CONNECTED)
+        if(status == WL_CONNECTED) {
             break;
-
-        //if 60 seconds then then activate AP
+        }
+        //if 60 seconds then then activate AP - TODO what's the 60 seconds? not seeing that being tracked anywhere here
```

Comment:
I picked 60 arbitrarily. We can set the SMWN timeout to something shorter if that works.